<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multiplication Table</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">

    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Select Level</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QComboBox" name="cb_level">
         <item><property name="text"><string>Easy</string></property></item>
         <item><property name="text"><string>Medium</string></property></item>
         <item><property name="text"><string>Hard</string></property></item>
         <item><property name="text"><string>Very Hard</string></property></item>
        </widget>
       </item>

       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="lbl_num1">
           <property name="font">
            <font><pointsize>50</pointsize><bold>true</bold></font>
           </property>
           <property name="alignment"><set>Qt::AlignCenter</set></property>
          </widget>
         </item>

         <item>
          <widget class="QLabel">
           <property name="font"><font><pointsize>50</pointsize></font></property>
           <property name="text"><string>x</string></property>
           <property name="alignment"><set>Qt::AlignCenter</set></property>
          </widget>
         </item>

         <item>
          <widget class="QLabel" name="lbl_num2">
           <property name="font">
            <font><pointsize>50</pointsize><bold>true</bold></font>
           </property>
           <property name="alignment"><set>Qt::AlignCenter</set></property>
          </widget>
         </item>
        </layout>
       </item>

      </layout>
     </widget>
    </item>

    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Your Answer</string>
      </property>
      <layout class="QGridLayout">

       <item row="0" column="0">
        <widget class="QLineEdit" name="txt_result">
         <property name="alignment"><set>Qt::AlignCenter</set></property>
        </widget>
       </item>

       <item row="0" column="2">
        <widget class="QLabel" name="lbl_result">
         <property name="font">
          <font><pointsize>18</pointsize><bold>true</bold></font>
         </property>
        </widget>
       </item>

      </layout>
     </widget>
    </item>

   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
